src/foam/u2/layout/Layouts.js

```diff
@@ -1,23 +1,207 @@
 foam.CLASS({
   package: 'foam.u2.layout',
-  name: 'RowLayout',
+  name: 'Rows',
   extends: 'foam.u2.Element',
+
+  requires: [
+    'foam.u2.layout.Row',
+  ],
+
+  documentation: `
+    A layout for a group of rows layed out vertically.
+  `,
+
   methods: [
     function add() {
-      console.log('Adding row!');
-      return this.SUPER.apply(this, arguments);
+      /**
+       * ! Trying to add a border like with Col breaks the row, we can look into later
+       * ! As card rows aren't probable right now
+       * Here we are checking for two cases
+       * 1. If a Row element is added, then call the super on it
+       * 2. If another element is added, then wrap it in a Row element with default configuration
+       */
+      [...arguments].forEach(value => {
+        if ( this.Row.isInstance(value) ) {
+          this.SUPER(value);
+        }
+        else {
+          this
+            .start(this.Row)
+                .add(value)
+            .end();
+        }
+      })
+      return this;
+    },
+
+    function initE() {
+      this.SUPER();
     }
   ]
 });
 
 foam.CLASS({
   package: 'foam.u2.layout',
-  name: 'ColumnLayout',
+  name: 'Row',
   extends: 'foam.u2.Element',
+
+  documentation: `
+    An individual column row within a group of rows.
+  `,
+
+  methods: [
+    function initE() {
+      this.style({ 'display': 'block' }) // more specificity should enforce ruling
```

Comment:
I don't think I understand this comment?
Not a big deal because I think we'll want to change up the styling of rows to be flex too cuz then we can reuse the AlignmentTypes enum.

Reply:
Refactored to be like Cols and Col using flex